library/std/tests/thread.rs

```diff
@@ -37,3 +37,11 @@ fn thread_local_containing_const_statements() {
     assert_eq!(CELL.get(), 1);
     assert_eq!(REFCELL.take(), 1);
 }
+
+#[test]
+// Include an ignore list on purpose, so that new platforms don't miss it
+#[cfg(not(any(target_os = "vxworks", target_os = "redox", target_os = "l4re")))]
```

Comment:
vxworks implements this now: https://github.com/rust-lang/rust/blob/8ed95d1d9e149b5242316c91b3849c58f8320470/library/std/src/sys/pal/unix/thread.rs#L491-L502